images/cross-cloud/sle-hpc/byos/profiles.yaml

```diff
@@ -19,4 +19,5 @@ profiles:
     description: GCE configuration
     include:
       - csp/gce/addon/gce-tools
+      - csp/gce/addon/hpc-tweaks
```

Comment:
hpc-tweaks??? really, how about `hpc-settings` or `hpc-configs`

Reply:
Well, it tweaks the GCE module for use with the HPC module. I've used the term before for CHOST on Azure, although admittedly, the HPC add-on for GCE is a bit more substantial. I can rename both.